SwiftyBluetooth/Source/Central.swift

```diff
@@ -62,6 +62,18 @@ public enum PeripheralScanResult {
 }
 
 /**
+ The different results returned in the closure of the Central connect(...) and disconnect(...) functions.
```

Comment:
@tehjord This looks interesting! Will likely pull this into our fork once it is merged to master.